plenum/test/view_change/test_view_change_timeout.py

```diff
@@ -10,62 +12,117 @@
 from stp_core.loop.eventually import eventually
 
 nodeCount = 7
-view_change_timeout = 5
+VIEW_CHANGE_TIMEOUT = 5
+
+
+@contextmanager
+def perf_monitor_disabled(tconf):
+    old_unsafe = tconf.unsafe.copy()
```

Comment:
Should we use `deepcopy` here? For the current example copy is enough, but if this becomes a generic method, deepcopy may be safer.

Reply:
Yes, I was thinking about writing generic context manager for temporary patching arbitrary attributes and for it `deepcopy` would me more appropriate, but in the end I think that it will be overkill, a small number of focused context managers with clear names look better (at least in my opinion). And in this case `copy` is enough.

Also I looked whether `deepcopy` could be used to "save" whole tconf, so context managers won't be necessary at all and it seems like it can fail in some cases. Root tconf calls global getConfig()  and returns it as is, without copying, so any changes in downstream tconfs will affect global config. Making it deepcopy global config as well might work, but we need to be sure that tconf is passed to every place where config is needed, and I'm not sure it's the case now.